<launch>
	<arg name="veh"/>
	<arg name="pkg_name" value="anti_instagram"/>
    <arg name="node_name" default="anti_instagram_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
    <param name="~veh" value="$(arg veh)"/>

	<group ns="$(arg veh)">
        <node name="$(arg node_name_name)" pkg="$(arg pkg_name)" type="$(arg node_name_name).py" output="screen">
            <rosparam command="load" file="$(find anti_anti_instagram)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
            <remap from="anti_instagram_node/uncorrected_image/compressed" to="camera_node/image/compressed"/>
        </node>
    </group>
</launch>
